hearth/lib/hearth/telemetry/context_manager.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Hearth
+  module Telemetry
+    # Base for all ContextManager classes
+    class ContextManager
+      def current; end
```

Comment:
These should probably have some level of documentation since this is part of the public interface.

Reply:
I think we should actually just delete this class. We don't have base classes anymore that are no-ops like retry strategy or signers.